src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2094,7 +2093,10 @@ void MKLDNNEltwiseNode::appendPostOpsImpl(mkldnn::post_ops& ops, const VectorDim
             }
             depthwiseDataSize = 2 * postOpDims[chIdx];
 
-            depthwiseData.resize(rnd_up(depthwiseData.size(), bufferAlignment), 0);
+            // always align for legacy scale/shift post ops
+            constexpr int bufferAlignment = 16;
+            int bufferPaddingSize = rnd_up(postOpDims[chIdx], bufferAlignment) - postOpDims[chIdx];
```

Comment:
Nit. We can introduce a named variable, e.g. `cannelsSize` or `seqLength = postOpDims[chIdx]` so this part of code become more clear for developers. The idea is to clearly emphasize that we need to pad data not to right align the whole buffer size but to align the processing sequence length to prevent out of bounds reading.

Reply:
I guess only new variable will not be enough here. Will add proper comment that describes the behavior in separate PR